app/code/Magento/Cms/Controller/Adminhtml/Page/Delete.php

```diff
@@ -15,6 +17,23 @@ class Delete extends \Magento\Backend\App\Action
      */
     const ADMIN_RESOURCE = 'Magento_Cms::page_delete';
 
+    /**
+     * @var \Magento\Cms\Model\Page
+     */
+    private $_cmsPage;
```

Comment:
Since this property is private we can rename it so that it does not include the underscore `$cmsPage` would be fine in this case.